krcal/map_builder/map_builder_functions.py

```diff
@@ -497,12 +493,13 @@ def remove_peripheral(map       : ASectorMap,
 
 def add_krevol(maps         : ASectorMap,
                dst          : pd.DataFrame,
+               masks_cuts   : masks_container,
```

Comment:
this parameter is not in docstrings, also why we dont have docstrings for all the functions in this file?

Reply:
It's a good question, maybe a new PR adding documentation. I did also now for the new function I added.